README.md

```diff
@@ -2,4 +2,6 @@
 
 **This is a fork of Facebook's [Create React App](https://github.com/facebook/create-react-app).**
 
-Please see refer to the `react-scripts` package [readme](packages/react-scripts/README.md) for details on how to use backpack-react-scripts.
+Files of interest:
```

Comment:
Maybe it just needs to say something like "this repo contains the following published packages",  with a link to each of them?

Reply:
The problem for me was that the repo is called "backpack-react-scripts", which is the name the package under `packages/react-scripts` is published under.

Reply:
Makes sense :)

Still think it'd be nice to at least mention that this repo contains multiple packages, for those not used to working in mono repos?